administrator/components/com_banners/tmpl/clients/emptystate.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_banners
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\Layout\LayoutHelper;
+
+$displayData = array(
+		'textPrefix' => 'COM_BANNERS_CLIENT',
+		'formURL'    => 'index.php?option=com_banners&view=clients',
+		'helpURL'    => 'https://docs.joomla.org/Special:MyLanguage/Help40:Banners:_Clients',
+		'icon'       => 'icon-bookmark banners',
+);
+$user        = Factory::getUser();
```

Comment:
Please dont use deprecated function calls. use 
```suggestion
$user        = Factory::getApplication()->getIdentity();
```